src/altinn-app-frontend/src/features/form/data/formDataSlice.ts

```diff
@@ -66,9 +66,6 @@ const formDataSlice = createSlice({
       state.isSaving = false;
       state.ignoreWarnings = true;
     },
-    // The _action parameter is unused, but these parameters are used by TypeScript to infer the payload type for
-    // this action, so we need to keep it here even if we only read it in the saga.
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This was actually meant to be short-lived, as it gets removed again in #336. In `createSagaSlice`, the API allows for entirely empty/non-reducer actions (without having to declare them elsewhere with `createAction()`), and it's also allowed for a reducer to ignore the payload even if the action is declared to expect a payload.

Reply:
I reverted this change already, based on your previous comments 🤓

Reply:
Oh no! My github incompetence is showing! :laughing: